### PR TITLE
DOCSP-40799: Projection guide

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-40799
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40799-projection/read/project/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by mongoKart — APPROVED** [review]
one small change, but lgtm otherwise!

File: source/includes/read/project.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#include <cstdint>
+#include <iostream>
+#include <vector>
```
Comment (kevinAlbs) [inline source/includes/read/project.cpp:3]:
```suggestion
#include <iostream>
```

Remove unused includes.

File: source/includes/read/project.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    mongocxx::options::find opts{};
+    opts.projection(make_document(kvp("_id", 0), kvp("name", 1), kvp("cuisine", 1), kvp("borough", 1)));
+
+    auto cursor = collection.find(make_document(kvp("name", "Emerald Pub")), opts);
+    for(auto&& doc : cursor) {
+        std::cout << bsoncxx::to_json(doc) << std::endl;
+    }
+    // end-project-include-without-id
+
+    // Retrieves documents matching the "name" field query and excludes their "grade" and "address" values when printing
```
Comment (kevinAlbs) [inline source/includes/read/project.cpp:47]:
```suggestion
    // Retrieves documents matching the "name" field query and excludes their "grades" and "address" values when printing
```

**Review by kevinAlbs — APPROVED** [review]
LGTM with minor comments.

**Review by Julia-Garland — APPROVED** [review]
Also LGTM!

**Pull request merged**